app-info/eos-extra.xml

```diff
@@ -190,11 +198,12 @@
     </categories>
   </component>
   <component type="desktop" merge="append">
-    <id>org.audacityteam.Audacity.desktop</id>
+    <id>org.audacityteam.Audacity</id>
     <kudos>
       <kudo>GnomeSoftware::popular</kudo>
     </kudos>
   </component>
+  <bundle type="flatpak"></bundle>
```

Comment:
Also added outside the component. It would help if the additions we're indented correctly.

Reply:
Oops, I should have caught this in my review. :(